src/controller/CHIPCluster.h

```diff
@@ -58,6 +59,9 @@ class DLL_EXPORT ClusterBase
     CHIP_ERROR AssociateWithGroup(DeviceProxy * device, GroupId groupId);
 
     void Dissociate();
+    // Temporary function to set command timeout before we move over to InvokeCommand
+    // TODO: remove when we start using InvokeCommand everywhere
```

Comment:
The TODO here probably involves adding a timeout param to InvokeCommand too?

Reply:
ack - they were there. Must have gotten blown away by the merge. I'll add that back. There's a whole commit that does nothing but deal with Invoke command plumbing that's just gone.

Reply:
Actually, wait - I just did it differently. I passed the value from mTimeout. So I suppose we could add it directly to the Invoke call. Up to the IM folks how they want to do that.

Reply:
https://github.com/project-chip/connectedhomeip/issues/13790 to track.